app/utils/ImplicitCurrencyFormatter.scala

```diff
@@ -26,6 +26,7 @@ trait ImplicitCurrencyFormatter {
     def toCurrency: Html = Html("&pound;" + f.format(x).replace(".00",""))
     def toCurrencyString: String = "£" + f.format(x).replace(".00","")
     def toStringNoDecimal: String = x.toString.replace(".0","")
+    def toPence: Int = x.toInt * 100
```

Comment:
This won't work as expected. You can only do the `.toInt` after the multiplication, otherwise you will lose the decimal places. See below:

```
scala> val x = BigDecimal(12.22)
x: scala.math.BigDecimal = 12.22

scala> x.toInt * 100
res0: Int = 1200

scala> (x*100).toInt
res1: Int = 1222
```